learning/intermediate/session-4.js

```diff
@@ -45,8 +45,24 @@ const person8 = null
 // IF they CAN drink then you will console log out there {toastMessage} AND THEN return TRUE
 // IF they can NOT drink, do nothing and return FALSE
 // keep in mind that it's POSSIBLE for the object to be empty or undefined, and therefore the properties {age} and {toastMessage} may NOT be defined
-function canPersonDrink() {
+function canPersonDrink(person) {
   // fill in the function here
+
+  const checkAge = person.age
+  const checkToastMsg = person.toastMessage
+  const validateAge = () => {
+    checkAge >= 21 && console.log(checkToastMsg)
+    return true
+  }
+
+  const falseReturn = () => {
```

Comment:
Why is a return statement enclosed in a function? You don't need this either. It's a simple as writing `return true` or `return false` in an if/else statement. 

Since you locked up your _if statement_ in the `validateAge` function, it's not longer useable outside the `validateAge` function. Rethink your approach.